src/main/java/com/minecolonies/rankup/modules/databases/command/convertToConfCommand.java

```diff
@@ -0,0 +1,96 @@
+import org.spongepowered.api.command.CommandSource;
+import org.spongepowered.api.command.args.CommandContext;
+import org.spongepowered.api.scheduler.Task;
+import org.spongepowered.api.text.Text;
+
+import java.util.Optional;
+import java.util.UUID;
+
+public class convertToConfCommand extends RankupSubcommand
+{
+    @Override
+    protected String[] getAliases()
+    {
+        return new String[] {"database-to-conf"};
+    }
+
+    @Override
+    protected Optional<Text> getDescription()
+    {
+        return Optional.of(Text.of("None"));
+    }
+
+    @Override
+    protected Optional<String> getPermission()
+    {
+        return Optional.empty();
+    }
+
+    @Override
+    public CommandResult execute(final CommandSource src, final CommandContext args) throws CommandException
+    {
+        Task.Builder taskBuilder = Sponge.getScheduler().createTaskBuilder();
+        taskBuilder.name("conf-to-database").async().delayTicks(4L).execute(this::convert).submit(getPlugin());
+
+        src.sendMessage(Text.of("Conversion has started, feel free to check your Console for progress!"));
+
+        return CommandResult.success();
+    }
+
+    private void convert()
+    {
+        AccountConfigData accConfig = getPlugin().configUtils.getAccountConfig();
+
+        int index = 0;
+
+        for (final UUID uuid : getPlugin().accUtils.getPlayers().keySet())
+        {
+            if (accConfig.playerData.containsKey(uuid))
+            {
+                AccountConfigData.PlayerConfig playerConfig = accConfig.playerData.get(uuid);
+
+                final String name = getPlugin().accUtils.getPlayerName(uuid);
+                final String joinDate = getPlugin().accUtils.getPlayerJoinDate(uuid);
+                final String lastDate = getPlugin().accUtils.getPlayerLastDate(uuid);
+                final int timePlayed = getPlugin().accUtils.getPlayerTime(uuid);
+
+                playerConfig.playerName = name;
+                playerConfig.joinDate = joinDate;
+                playerConfig.lastVisit = lastDate;
+                playerConfig.timePlayed = timePlayed;
+            }
+            else
+            {
+                accConfig.playerData.put(uuid, new AccountConfigData.PlayerConfig());
+
+                AccountConfigData.PlayerConfig playerConfig = accConfig.playerData.get(uuid);
+
+                final String name = getPlugin().accUtils.getPlayerName(uuid);
+                final String joinDate = getPlugin().accUtils.getPlayerJoinDate(uuid);
+                final String lastDate = getPlugin().accUtils.getPlayerLastDate(uuid);
+                final int timePlayed = getPlugin().accUtils.getPlayerTime(uuid);
+
+                playerConfig.playerName = name;
+                playerConfig.joinDate = joinDate;
+                playerConfig.lastVisit = lastDate;
+                playerConfig.timePlayed = timePlayed;
+            }
+            index++;
+
+            if (index % 25 == 0)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 25 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS109)